src/fu-util-common.c

```diff
@@ -1633,7 +1633,7 @@ fu_util_security_attrs_to_string (GPtrArray *attrs, FuSecurityAttrToStringFlags
 
 			/* check for PCR0 not matching */
 			if (g_strcmp0 (fwupd_security_attr_get_appstream_id (attr),
-					FWUPD_SECURITY_ATTR_ID_TPM_RECONSTRUCTION_PCR0) == 0 &&
+					FU_SECURITY_ATTR_ID_TPM_RECONSTRUCTION_PCR0) == 0 &&
```

Comment:
So I think this specific one maybe we should stop special casing the client messaging.  It can be punted into https://github.com/fwupd/fwupd/wiki/Low-host-security-level and that page can link to  https://github.com/fwupd/fwupd/wiki/TPM-PCR0-differs-from-reconstruction

Then there is no longer a client dependency on attributes

Reply:
Although I guess you still have `fu_security_attr_get_name` is used.. Hmm

Reply:
I guess one approach is to rather than offer translations in the client by `fu_security_attr_get_name` to just leave appstream IDs.  Let the appstream ID decoder live in HSI docs (when they are live) or in wiki pages.

Reply:
>  I think this specific one maybe we should stop special casing the client messaging

Do you think now the PCR0 calculation has been fixed in fwupd we'll stop getting this? I still think it's somewhat useful to have translated content to cover the special cases.

Reply:
I would like to think so, but we'll truly find out later. It's always a revert away to add/remove translations. How about for now the command line client uses the private file to get those strings and we can decide again later whether they stay.

Reply:
The fwupd-security-attr-private.h can't provide translations, it gets really messy when dealing with all the msgcontext stuff in a library.

> Then there is no longer a client dependency on attributes

I think fwupdmgr and fwupdtool can be considered internal enough, or are you worried about the case where a new daemon uses org.dtmf.hsi prefixes and the old client is expecting org.fwupd.hsi prefixes?

Reply:
Oh I mean leave translations as is and just include the private header for now. We can remove them later when hsi goes stable if we decide that makes sense.

So yes keep it treated as internal for now.

Reply:
> and just include the private header for now

That's what I did, no?

Reply:
I don't know, you haven't pushed anything :)

Reply:
Dude, you need me to do everything! :) Pushed now, sorry about that.